handler/mail.go

```diff
@@ -82,3 +92,10 @@ func encodeRFC2047(String string) string {
 	addr := mail.Address{Name: String, Address: ""}
 	return strings.Trim(addr.String(), "<@>")
 }
+
+func MergeSlice(s1 []string, s2 []string) []string {
```

Comment:
Use `append` method in the standard library and `...` operator instead.
You can find references at https://www.geeksforgeeks.org/how-to-append-a-slice-in-golang/

Don't write a method that already exists in the standard library